react-native-libraries.json

```diff
@@ -1733,5 +1747,33 @@
     "android": true,
     "web": false,
     "expo": false
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-paper",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-fbads",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-material-palette",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-opentok",
```

Comment:
This library is a little dated, have you used it? If you think it's good I'll take your word for it.

Reply:
Actually I'm not using this library. So I don't known that this library is good or bad. If you don't need this library I'll remove it. :+1:

Reply:
@ubatin so for the future we should only add libraries we personally like or use. That helps us keep the qualities of libraries listed above average.

Reply:
lets remove this one.

Reply:
@jimmylee , thank your review. I'll remove this library.

Reply:
hi @jimmylee , I removed this library :tada: